packages/extension-ui/src/util/openInExternalPopup.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+const EXTERNAL_POPUP_PATH = 'notification.html';
+
+export default (path: string) => {
```

Comment:
the window doesn't open in firefox, I can't figure out why :disappointed: 
errors I get:
```
Error: Please use $(ref:runtime.getURL). [extension.js:21973](moz-extension://56639082-6668-4ead-8a52-275009c3ac8b/extension.js)
    __WEBPACK_DEFAULT_EXPORT__ moz-extension://56639082-6668-4ead-8a52-275009c3ac8b/extension.js:21973
```
```
WebExtension context not found! [ExtensionParent.jsm:1296](resource://gre/modules/ExtensionParent.jsm)
    getContextById resource://gre/modules/ExtensionParent.jsm:1296
    recvAPICall resource://gre/modules/ExtensionParent.jsm:1133
    _recv resource://gre/modules/ConduitsChild.jsm:84
    receiveMessage resource://gre/modules/ConduitsParent.jsm:466
```

The find one point at this line:
```
url: chrome.extension.getURL(`${EXTERNAL_POPUP_PATH}#/${path}`),
```
perhaps it's somehow related to `ALLOWED_PATH`?

```
packages/extension-base/src/defaults.ts:17
```

:sweat_smile:

Reply:
Yup, using the background message fixed it. Great finding 🎖️

Reply:
Also, btw - before applying the fix, I haven't seen any logs while debugging this in FF (but I did encounter the problem with the window not opening). Where did you get them from? The debugging extension you told me once about?

Reply:
![image](https://user-images.githubusercontent.com/35614528/236473865-72ba913d-9bf1-4451-92d1-cf0eb8a03215.png)
![image](https://user-images.githubusercontent.com/35614528/236474243-c732a79b-054a-4833-9c32-6dfb47386dea.png)
![image](https://user-images.githubusercontent.com/35614528/236474338-70e736bf-9276-4c9a-af6d-2d1c75531f5f.png)
there is another way:
![image](https://user-images.githubusercontent.com/35614528/236474615-6c01001f-874a-47c9-a3a6-03128a32da02.png)